dist/templates/ovnkube-zone-controller.yaml.j2

```diff
@@ -442,6 +446,12 @@ spec:
       - name: host-etc-ovs
         hostPath:
           path: /etc/openvswitch
+      {% if enable_ipsec=="true" -%}
+      - name: run-pluto
+        hostPath:
+          path: /var/run/pluto
+          type: DirectoryOrCreate
```

Comment:
would it be already created prio to ovnk coming up? can we fail the directory doesnt exist instead of creating?